javaxval/src/main/resources/META-INF/services/javax.validation.valueextraction.ValueExtractor

```diff
@@ -1 +1 @@
-org.baeldung.valueextractors.ProfileValueExtractor
\ No newline at end of file
+org.baeldung.javaxval.container.validation.valueextractors.ProfileValueExtractor
```

Comment:
@lor6 This exact fragment is used in an article, do we want to update it?

Reply:
yep, I see the package is specifically mentioned in the article; I'll add a note in the JIRA to update it after the merge

Reply:
Article: https://www.baeldung.com/bean-validation-container-elements

Search for `Finally, we need to register the class`